build.gradle

```diff
@@ -29,6 +29,23 @@ allprojects {
     }
 
     version gitVersion()
+
+    if (System.env.CIRCLE_ARTIFACTS) {
+        task copyCiArtifactsToCircle {
```

Comment:
Should I see `copyCiArtifactsToCircle` in `./gradlew tasks`?

On Thu, Jul 14, 2016 at 6:17 PM gracew <notifications@github.com> wrote:

> In build.gradle
> <https://github.com/palantir/auth-tokens/pull/15#discussion_r70894224>:
>
> > @@ -29,6 +29,23 @@ allprojects {
> >      }
> >
> >      version gitVersion()
> > +
> > +    if (System.env.CIRCLE_ARTIFACTS) {
> > +        task copyCiArtifactsToCircle {
>
> we need to call this task somewhere in circle.yml
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/auth-tokens/pull/15/files/fb824d4a3553669de26275df64abdd0f228c927f#r70894224>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC0hDUOBRG1tnn38fcT6Aqy6BuxuKeSgks5qVrVsgaJpZM4JMmHT>
> .
>
-- 
Andrew